### ISSUE TITLE
Jupyter notebook unable to launch after certificate installs

### ISSUE THREAD

**Issue by paytonong** [jupyter/notebook#2573]
Hi all, I'm a novice linux user and no problems installing and using jupyter notebook.  However, after installing Citrix receiver and some certificate permissions, I get the below error when I try to launch Jupyter Notebook.:

`PermissionError: [Errno 13] Permission denied: '/usr/local/lib/netscape/mime.types'`

After installing Citrix, I ran the below commands to since I was getting certificate errors when launching citrix from chrome. Notebooks that I had open crashed after I ran the updates for the certificates and I get the error above when I try to launch new ones. 

```
sudo update-ca-certificates 
sudo ln -s /usr/share/ca-certificates/mozilla/* /opt/Citrix/ICAClient/keystore/cacerts/
sudo c_rehash /opt/Citrix/ICAClient/keystore/cacerts/
/opt/Citrix/ICAClient/util/configmgr &

```

Any help would be appreciated. Thanks.

**Comment by paytonong** [comment]
Any ideas on what the problem is?

**Comment by takluyver** [comment]
No, I don't know anything about Citrix receiver, nor what code might be trying to open `/usr/local/lib/netscape/mime.types`. Sorry.

**Comment by hakimkhalafi** [comment]
Hi, I had the exact same problem. The fix is a change of ownership for the file

`cd /usr/local/lib/netscape/
sudo chgrp <user> mime.types
sudo chown <user> mime.types
`
where <user> is your username
